deployment/.template.env

```diff
@@ -3,6 +3,17 @@
 # The name the app will be saved under in Docker compose
 COMPOSE_PROJECT_NAME=radixdlt-network-gateway__toy-deployment
 
+########################################################
+# DOCKER IMAGE VERSION TAGS & DEVELOPMENT DEPENDENCIES #
+########################################################
+
+FULLNODE_DOCKER_TAG=2.0.0 # TODO not published yet
+POSTGRES_DB_DOCKER_TAG=14.3-alpine
+NETWORK_GATEWAY_DOCKER_TAG=2.0.0 # TODO not published yet
+
+# Git commit version of most recent Node compatible with this Network Gateway (only used in CI/CD environments)
+FULLNODE_VERSION=cf1fc5657e42f3f6239c10555ff847cc856f40a9
```

Comment:
This is in the `.template.env` file, not the `.env` file - so I think this is wrong.

Each developer and CI should have all of these automatically update when they pull in the latest version.

I would suggest therefore that these live in a `versions.env` file at the root, and that this env file is loaded by the scripts automatically...

And we leave this `.template.env` for defaults to the _changes/overrides_ that users may wish to make themselves.

Reply:
Although I don't know if docker compose can handle that.

It might be that we need to create a `.env` file on the fly by combining the `/versions.env` with a `/deployment/settings.env` file to create a temporary `.env` file which is then used by docker compose?

Reply:
I imagine the Gateway ephemeral environment stuff will also want to read these, so having them in a separate `/versions.env` file will help them read them.

Reply:
> Each developer and CI should have all of these automatically update when they pull in the latest version.

That's precisely what we want NOT to happen, we want to control this Node version manually (just as say NuGet packages)

> Although I don't know if docker compose can handle that.

It can't, this is why I've decided to place all of that in the `.template.env` file which I wanted our Ephemeral envs to use. Why? To reduce number of extra scripts/steps to execute in order to just start all of that. 
Have I essentially piggybacked some CI-specific logic into Docker-specific file? Yes, and I think as long as it is literally single variable this should be fine. Obviously I could move this single variable (`FULLNODE_VERSION`) into `development/ci.env` or something.